wp-content/themes/reconasia/assets/_scss/layout/_footer.scss

```diff
@@ -59,67 +62,94 @@
 
   &__logo {
     grid-area: logo;
-    max-width: 300px;
+    margin-bottom: rem(16);
+  }
 
-    &,
-    &:visited {
-      // color: $color-text-gray-10;
-    }
+  &__desc {
+    grid-area: csisdesc;
+    margin-bottom: rem(40);
   }
 
   .widget:first-of-type {
-    grid-area: csisdesc;
+    margin-bottom: rem(24);
   }
 
-  .widget:nth-of-type(2) {
-    grid-area: magdesc;
+  &__desc + .widget {
+    grid-area: newsletter;
   }
 
-  &__contact {
+  &__social {
     display: grid;
-    grid-area: contact;
+    grid-area: social;
     grid-row-gap: rem(16);
     align-content: start;
-    margin-top: rem(16);
-    padding-top: rem(16);
-    border-top: 1px solid $color-white-130;
+    margin-top: rem(32);
+    padding-top: rem(24);
+    border-top: 1px solid $color-border-light-115;
 
-    @include breakpoint('medium') {
+    @include breakpoint('small') {
       margin-top: 0;
       padding-top: 0;
       padding-left: rem(16);
       border-top: 0;
-      border-left: 1px solid $color-white-130;
+      border-left: 1px solid $color-border-light-115;
+    }
+
+    @include breakpoint('medium') {
+      padding-left: rem(24);
     }
   }
 
-  &__address {
-    font-style: normal;
+  &__facebook-icon .icon,
+  &__twitter-icon .icon,
+  &__linkedin-icon .icon,
+  &__instagram-icon .icon,
+  &__youtube-icon .icon {
+    width: rem(28);
+    height: rem(42);
+    color: $color-white-165;
+  }
+
+  &__social-icons {
+    margin-top: rem(-16);
```

Comment:
Instead of a negative margin here, I would adjust the `grid-row-gap` property in `&__social` instead. That makes it consistent as to what property is controlling the space.